taskvine/src/bindings/python3/ndcctools/taskvine/compat/dask_executor.py

```diff
@@ -128,14 +128,14 @@ def get(self, dsk, keys, *,
             lib_modules=None,
             task_mode='tasks',
             scheduling_mode='FIFO',
+            worker_scheduler="schedule_files",
```

Comment:
Let's rename this one:  `schedule_tasks_by=`. Otherwise it may be confusing in that people may think we are scheduling the workers themselves. Also, given #4188 you will only need to pass "files", "random", etc.